Tests/SwiftDocCTests/Infrastructure/NodeURLGeneratorTests.swift

```diff
@@ -77,6 +77,19 @@ class NodeURLGeneratorTests: XCTestCase {
         XCTAssertEqual(generator.urlForReference(classIdentifier).absoluteString, "file:///path/to/bundle/_testbundle-ctlj/products/documentation.builtbundle/com.example.testbundle/data/folder/_privateclass/_privatesubclass")
     }
     
+    func testsafeURLWithEncodableFragment() throws {
```

Comment:
I don't think we want this test because we can't verify that it's created the resolved topic reference in the same way that the context is. This means that this test could pass while the end-to-end experience is still broken.

Reply:
Alright, I've removed it.